airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -473,20 +497,21 @@ def create_external_table(self,
     def run_query(self,
                   bql=None,
                   sql=None,
-                  destination_dataset_table=False,
+                  destination_dataset_table=None,
                   write_disposition='WRITE_EMPTY',
                   allow_large_results=False,
                   flatten_results=False,
-                  udf_config=False,
+                  udf_config=None,
                   use_legacy_sql=None,
                   maximum_billing_tier=None,
                   maximum_bytes_billed=None,
                   create_disposition='CREATE_IF_NEEDED',
                   query_params=None,
                   labels=None,
                   schema_update_options=(),
-                  priority='INTERACTIVE',
-                  time_partitioning=None):
+                  priority=None,
```

Comment:
I would not change the default value here, as it is going to show in autocompletion.